README.md

```diff
@@ -30,54 +28,57 @@ $ pip install meeshkan
 
-def read_http_exchanges() -> List[HttpExchange]:
-    """Read HTTP exchanges from the source of your choice.
-    """
-    request: Request = RequestBuilder.from_url(
-        "https://example.org/v1", method=HttpMethod.GET, headers={})
+### Hello world
 
-    response: Response = ResponseBuilder.from_dict(dict(
-        statusCode=200, body=json.dumps({"hello": "world"}), headers={}))
-
-    http_exchange: HttpExchange = HttpExchange(request=request, response=response)
+```bash
+$ meeshkan record -r --daemon # start the recorder in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # record traffic
+$ meeshkan record --stop-daemon # stop the daemon
+$ meeshkan build
+$ meeshkan mock -r --daemon # start the mocking server in daemon mode
+$ curl http://localhost:8000 -H '{"Host": "time.jsontest.com" }' # mock traffic
+$ meeshkan mock --stop-daemon # stop the daemon
+```
 
-    return [http_exchange]
+## Collect recordings of API traffic
 
+To build a Meeshkan spec, you must first **collect** recordings of server traffic and/or OpenAPI server specs.
 
-http_exchanges = read_http_exchanges()
+To record API traffic, the Meeshkan CLI provides a `record` mode that captures API traffic using a proxy.
 
-# Build OpenAPI schema from a list of recordings
-openapi: OpenAPIObject = build_schema_batch(http_exchanges)
+```bash
+$ pip install meeshkan # if not installed yet
+$ meeshkan record
+```
 
-# Build schema from an iterator
-openapi: OpenAPIObject = build_schema_online(iter(http_exchanges), UpdateMode.GEN)
+This starts Meeshkan as a reverse proxy on the default port of `8000`.  For example, with curl, you can use Meeshkan as a proxy like so.
 
-# Update OpenAPI schema one `HttpExchange` at a time
-http_exchange = http_exchanges[0]
-openapi: OpenAPIObject = update_openapi(openapi, http_exchange, UpdateMode.GEN)
+```bash
+$ curl http://localhost:8000/http://api.example.com
 ```
 
-## Builder
+By default, the recording proxy treats the path as the target URL and writes a [`.jsonl`](https://jsonlines.org) file containing logs of all server traffic to a `logs` directory.  All logs are created in the [`http-types`](https://github.com/meeshkan/http-types) format.  The `meeshkan build` tool expects all recordings to be represented in a `.jsonl` file containing recordings represented in the `http-types` format.
+
+For more information about recording, including direct file writing and kafka streaming, see the [recording documentation](./guides/RECORD.md).
+
+## Build a Meeshkan spec from recordings
 
-Using the Meeshkan CLI, you can build OpenAPI schema from a single `recordings.jsonl` file in the [HTTP Types](https://meeshkan.github.io/http-types/) JSON format.
+Using the Meeshkan CLI, you can **build** an OpenAPI schema from a single `.jsonl` file in the [HTTP Types](https://meeshkan.github.io/http-types/) JSON format in addition to any existing OpenAPI specs that describe how a servie works.
```

Comment:
```suggestion
Using the Meeshkan CLI, you can **build** an OpenAPI schema from a single `.jsonl` file, in addition to any existing OpenAPI specs that describe how a service works.
```

We talk about HTTP Types in the next paragraph so I think it's fine to omit here.